.gitpod.yml

```diff
@@ -1,3 +1,5 @@
+image: gitpod/workspace-full
```

Comment:
To show the demo of the image as we do in: https://github.com/gitpod-demos/voting-app/blob/main/.gitpod.yml